### PR TITLE
Allow user to resize glyphs

### PR DESCRIPTION
**Pull request by bnmajor**
Addresses #671

### CODE REVIEW

**Comment by bnmajor** [comment]
This currently uses a slider to adjust the size, but could easily be updated to use a spinbox instead if that makes more sense.

glyph_size

**Review by cjh1 — APPROVED** [review]

File: hexrd/ui/indexing/fit_grains_results_dialog.py [hunk 1]
```diff
@@ -442,6 +445,12 @@ def update_range_constraints(self):
             w1.setMaximum(w2.value())
             w2.setMinimum(w1.value())
 
+    def set_glyph_size(self, reset=True):
```
Comment (psavery) [inline hexrd/ui/indexing/fit_grains_results_dialog.py:448]: Maybe this should be called `reset_glyph_size`? With the `reset` argument being called `update_plot`?

**Comment by psavery** [comment]
Can we also add tab stops for the new widgets?

**Comment by psavery** [comment]
I tested it out and works well for me.

File: hexrd/ui/indexing/fit_grains_results_dialog.py (hunk 1 above)
Comment (bnmajor) [reply hexrd/ui/indexing/fit_grains_results_dialog.py]: Sounds good, makes it clearer. I've updated the name of the function and parameter.

**Comment by bnmajor** [comment]
> Can we also add tab stops for the new widgets?

Updated the tab stops - Thanks for pointing that out, @psavery!

**Review by psavery — APPROVED** [review]
Thanks @bnmajor! Looks good to me.

**Pull request merged**